manager/scheduler/filter.go

```diff
@@ -169,7 +169,7 @@ func (f *PluginFilter) Check(n *NodeInfo) bool {
 		}
 	}
 
-	if f.t.Spec.LogDriver != nil {
+	if f.t.Spec.LogDriver != nil && f.t.Spec.LogDriver.Name != "none" {
```

Comment:
@cpuguy83 do we need to check for "None" too or is this a case-sensitive keyword?

Reply:
We could check lower, but it is case sensitive in docker at least.

Reply:
Okay, in that case we can leave it as is for now.

Reply:
Also wondered if `nil` and `"none"` are the same, but I think they have a different meaning (`nil` being: use the default?) If not, and the _are_ equal then ideally we should change `"none"` to `nil` in the spec